backend/src/email/middlewares/email-transactional.middleware.ts

```diff
@@ -25,6 +25,14 @@ export interface EmailTransactionalMiddleware {
   rateLimit: Handler
 }
 
+export const RATE_LIMIT_ERROR_MESSAGE =
+  'Error 429: Too many requests, rate limit reached'
+
+export const getMd5HashFromAttachment = (content: Buffer): string => {
```

Comment:
Why are we changing this function name? Adding `Md5` and `From` doesn't provide any extra context. Because for the reader, I don't need to care about the underlying algo and preposition

Reply:
Also, please don't export this function, u can just mock the underlying `crypto` package. This function is intentionally kept private

Reply:
> Also, please don't export this function, u can just mock the underlying crypto package. This function is intentionally kept private

Ohh that's a very good point, let me change it back.

Can you show me how to mock the underlying `crypto` package? I think I have seen you do it in a previous PR before, but I don't fully understand how that works.

Reply:
Use [this](https://jestjs.io/docs/manual-mocks)

Reply:
Unresolved comment